speaker/main.go

```diff
@@ -158,6 +159,9 @@ func main() {
 		SList:                  sList,
 		bgpType:                bgpImplementation(bgpType),
 		InterfaceExcludeRegexp: interfacesToExclude,
+		Layer2StatusChange: func(svc *v1.Service) {
```

Comment:
I think it's cleaner if this would receive a namespacedname instead of a service (because we don't do anything meaningful with the service object)